ecommerce_extensions/core/utils.py

```diff
@@ -0,0 +1,13 @@
+"""Utils module, common project methods."""
+from django.conf import settings
+
+
+def has_console_access(user):
+    """Check if user has access to extra content."""
+    if user.is_staff:
+        return True
+
+    if not getattr(settings, 'ALLOW_CONSOLE_ACCESS', False):
+        return False
+
+    return user.groups.filter(name='ecomm_console_client').exists()
```

Comment:
@andrey-canon  Why is it necessary to also add the setting 'ALLOW_CONSOLE_ACCESS'? Wouldn't be enough to just check that the user belongs to the _ecomm_console_client_ group?
Everything else LGTM

Reply:
for example you have  10000000 users and then you give access to 15 users on site A and one year later you want to remove this from site A, you don't remember those users,  so you have to do a special query to look for those users, this site setting avoids that.

Reply:
great use case, sir